gadget/install/partition.go

```diff
@@ -177,42 +177,55 @@ func deviceName(name string, index int) string {
 
 // removeCreatedPartitions removes partitions added during a previous install.
 func removeCreatedPartitions(lv *gadget.LaidOutVolume, dl *gadget.OnDiskVolume) error {
-	indexes := make([]string, 0, len(dl.Structure))
+	sfDiskindexes := make([]string, 0, len(dl.Structure))
```

Comment:
nitpick: it's not clear what `sf` stands for, and the capitalization on `indexes` is missing.

Reply:
well I meant for this to be `sfdiskIndexes` since these are indexes to use with sfdisk below but that clearly is not what I wrote :-)

Reply:
done